index.html

```diff
@@ -11,18 +11,29 @@
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600;700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/db2c8620f4.js" crossorigin="anonymous"></script>
 </head>
 <body>
-  <header class="presentation">
+  <div></di>
+  <header id="presentation">
     <nav class="nav-container">
       <img class="my-logo logo-animation" src="Images/Logo HG.png">
-      <i style="color: #acb7c3" class="fa fa-bars desktop-hide"></i>
-      <ul class="mobile-hide nav-items-list">
-        <li>Portfolio</li>
-        <li>About</li>
-        <li>Contact</li>
+      <div class="burguer">
+        <ul id="hamburguer">
+        <span class="ham-bar"></span>
+        <span class="ham-bar"></span>
+        <span class="ham-bar"></span>
+        </ul>
+      </div>
+      <div class="close-icon"><i class="fa-solid fa-xmark blur"></i></div>
+      <ul class="nav-items-list menu-list desktop-shown">
+        <li><img src="Images/Closing-cross.png" class="cross close-cross"></li>
+        <li><a href="#card-container" class="portfolio">Portfolio</a></li>
+        <li><a href="#about-myself" class="about">About</a></li>
+        <li><a href="#contact-me" class="contact">Contact</a></li>
       </ul>
```

Comment:
- Kindly make sure the mobile menu is not scrollable, and your mobile will not be open after the first time, please fix it to have a better user experience, look at the following video to understand what I mean.
https://www.loom.com/share/93ccd3611baa4285ad7bcca65121da4f 👍